launchflow/aws/lambda_function.py

```diff
@@ -72,6 +72,9 @@ class LambdaFunctionInputs(ResourceInputs):
     memory_size_mb: int
     package_type: Literal["Image", "Zip"]
     runtime: Optional[LambdaRuntime]
+    vpc: bool
```

Comment:
super nit: slight preference for "use_vpc" instead of "vpc" since I think we'll eventually want a way to pass in a VPC object, so might make the migration easier if we leave the "vpc" param open

Reply:
chatted offline and decided to leave it, we can always overload this to allow supporting specific vpc